html

KSEN0X opened the pull request:
html markup

KSEN0X committed:
html

KSEN0X committed:
html: fixed div and comments

KSEN0X committed:
Merge remote-tracking branch 'origin/main' into recommendations

Hellwood1 commented on src/partials/recomendation.html:
```diff
@@ -1 +1,55 @@
 <!-- <section class="recomendation-section"> Оксана</section> -->
```
Видали закоментовану частину коду, яка не використовується.

KSEN0X replied:
Зроблено!

Hellwood1 commented on src/partials/recomendation.html:
```diff
@@ -1 +1,55 @@
+      </picture>
+    </div>
+    
+    <!-- --------------------------------------------------- -->
```
Валідатор ругається на закоментовані риски, які слугують в ролі роздільника, тому їх краще убрати, або ж відредагувати!

KSEN0X replied:
Видалила!

Hellwood1 commented on src/partials/recomendation.html:
```diff
@@ -1 +1,55 @@
+      </p>
+    </div>
+    </div>
+    </div>
```
Лишній закриваючий тег </div>

KSEN0X replied:
Видалила!

Hellwood1 reviewed (commented):
Привіт Оксано!

Я не можу приняти цей реквест, оскільки розмітка має декілька вад. 
- Викачай картинки в png форматі, що дозволить убрати білі кутки;
- Оптимізуй зоображення через squosh або інші сторонні ресурси;
- Виправ моменти, які я описав в коментарях та прожени свою розмітку через Validator, бо він видає помилки! 
Виправ ці не значні вади, та надішли мені код на перевірку! 

Буду чекати!

KSEN0X commented:
Перевірено на валідаторі !!!

Hellwood1 committed:
Merge branch 'recommendations' of https://github.com/Hellwood1/energy-flow-team-project into recommendations

Hellwood1 committed:
Update recomendation.html

Hellwood1 committed:
Changed some moments

- Змінив розташування картинок та видалив не діючу дерикторію;

- Виправив конфлікти;

- Теоретично, виправив шлях до відображення SVG

- Змінив заголовок з h2 на h3

Hellwood1 reviewed (approved):
Вніс маленьки корективи, але в цілому добре! 
Очікую на .css! 

Зверни увагу, що відступи між секціями мають бути реалізовані за допомогою ділення padding top and bottom пополам. 

Наприклад, якщо відступ від верхньої секції складає 16px на ПК версії, то в тегі секції, тобі потрібно встановити padding-top: 8px
І аналогічно з відступом від футеру. 

І так, на кожному девайсі!
